packages/@o3r/eslint-config-otter/jasmine.js

```diff
@@ -0,0 +1,11 @@
+/* eslint-disable @typescript-eslint/naming-convention */
+/* eslint-disable quote-props */
+module.exports = {
```

Comment:
Should we really provide this?
- we do not use Jasmine
- we do not recommend Jasmine
- this is just extending the recommended rule that can be done by the user directly
- this will require our generator to be adapted to include or not this file

I would actually not provide it.
The split index.js/base.js would be enough for me.

Reply:
yet we still provide services fixtures for Jasmine.
it doesn't cost much to export it in this package that doesn't change very often

> this will require our generator to be adapted to include or not this file

same applies to the `index/base` split that you suggested.

Reply:
Then fine for me as long as it is optional peerDeps 🙂